include/deal.II/lac/linear_operator.h

```diff
@@ -1118,6 +1371,228 @@ linear_operator(const OperatorExemplar &operator_exemplar, const Matrix &matrix)
 
 //@}
 
+namespace SchurComplement
```

Comment:
I think that remodeling those functions as a ```PackagedOperation``` does not require a ```Payload```, does it?
After all, the mechanism is very similar to the ```block_back_substitution``` we already have and the (not yet merged) ```constraint_linear_operator``` that can be fully described in terms of ```LinearOperator``` and ```PackagedOperation```